files/image_config/caclmgrd/caclmgrd

```diff
@@ -378,9 +378,7 @@ class ControlPlaneAclManager(object):
         # add iptables/ip6tables commands to drop all other incoming packets
         if num_ctrl_plane_acl_rules > 0:
             iptables_cmds.append("iptables -A INPUT -j DROP")
```

Comment:
Just thinking, does the INPUT rule drop the "forwarding" packets?

Reply:
tested this, and INPUT rule drop does not affect the "forwarding" packet. So we should be good